### ISSUE TITLE
Problem to remove end of slash

### ISSUE THREAD

**Issue by voltan** [pi-engine/pi#1527]
Hello

after remove end of slash from `Pi/Mvc/Route/Standard.php` we have some problems on main templates , some urls not work , for example on system module rule and nav pages , I add pull request to revert it as temporary solution , do you have any plan for fix it ? 

what is your idea?

**Comment by Marc-pi** [comment]
yes, https://github.com/pi-engine/pi/issues/1524#issuecomment-288020777
